src/Entity/DeliveryConfirmation.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Ups\Entity;
+
+use DOMDocument;
+use DOMElement;
+use Ups\NodeInterface;
+
+/**
+ * Class DeliveryConfirmation
+ * @package Ups\Entity
+ */
+class DeliveryConfirmation implements NodeInterface
+{
+    /**
+     * @var
+     *
+     * 1 => 'Delivery Confirmation Signature Required'
+     * 2 => 'Delivery Confirmation Adult Signature Required.'
+     */
+    private $dcisType = 1;
```

Comment:
Not sure we need a default value in here, WDYT? Might give people errors when using it unaware of this.